node/Cargo.toml

```diff
@@ -19,11 +19,12 @@ targets = ['x86_64-unknown-linux-gnu']
 substrate-build-script-utils = '3.0.0'
 
 [dependencies]
+funty = "1.1.0"
```

Comment:
Better to align with parachain repo
Currently parachain repo uses command to fix this in cargo.lock
Perhaps cargo update, I forget the name...

Reply:
will remove it and use the command to update lock file.